nat/src/main/java/org/hyperledger/besu/nat/core/domain/NatPortMapping.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.hyperledger.besu.nat.core.domain;
+
+/** This class describes a NAT configuration. */
+public class NatPortMapping {
+
+  private final NatServiceType natServiceType;
+  private final NetworkProtocol protocol;
+  private final String internalHost;
+  private final String remoteHost;
+  private final int externalPort;
+  private final int internalPort;
+
+  public NatPortMapping(
+      final NatServiceType natServiceType,
+      final NetworkProtocol protocol,
+      final String internalHost,
+      final String remoteHost,
+      final int externalPort,
+      final int internalPort) {
+    this.natServiceType = natServiceType;
+    this.protocol = protocol;
+    this.internalHost = internalHost;
+    this.remoteHost = remoteHost;
+    this.externalPort = externalPort;
+    this.internalPort = internalPort;
+  }
+
+  /**
+   * Builds an empty representation of a {@link NatPortMapping}
+   *
+   * @param protocol the {@link NetworkProtocol} associated to the representation.
+   * @return the built {@link NatPortMapping}
+   */
+  public static NatPortMapping emptyPortMapping(final NetworkProtocol protocol) {
+    return new NatPortMapping(null, protocol, "", "", 0, 0);
+  }
```

Comment:
My intellij tells me this isn't used anywhere, I'm guessing it's for the next PR?

Reply:
I deleted this method and I will add it when it will be necessary